tests/unit/query_graph/conftest.py

```diff
@@ -357,6 +357,29 @@ def query_graph_with_groupby_fixture(query_graph_and_assign_node, groupby_node_p
     return graph
 
 
+@pytest.fixture(name="query_graph_with_forward_aggregate")
+def query_graph_with_forward_aggregate_fixture(query_graph_and_assign_node, event_table_details):
+    """Fixture of a query graph with a forward aggregate operation"""
+    graph, assign_node = query_graph_and_assign_node
+    graph.add_operation(
+        node_type=NodeType.FORWARD_AGGREGATE,
+        node_params={
+            "name": "target_node",
```

Comment:
nit: Might be a bit less confusing to use an actual target name here because it is unrelated to graph or nodes.